### ISSUE TITLE
clang-10 tests for stable branches?

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/continuous-integration2#279]
https://github.com/ClangBuiltLinux/continuous-integration2/pull/278 spun down our remaining clang-10 tests.  I'm curious if it's worthwhile to keep some arm64 + x86_64 builds going for branches of stable with clang-10?  We can probably look and see what clang versions various Android releases are using.

**Comment by nathanchance** [comment]
I was thinking more about this within the context of #278 and I am not sure there is much value in continuing to build with `clang-10`.

* We don't have any avenue to fix bugs in the compiler because it is not supported anymore (one of the main reasons we bumped to `clang-11` on mainline).
* If something breaks in stable for one reason or another, inserting workarounds into the source is not going to be easy without them being upstream (the stable hates taking patches just for stable).

**Comment by nickdesaulniers** [comment]
Right, but clang-10 was supported when those branches of stable were created, and Android is still using clang-10 based versions of AOSP Clang.  Having some heads up when stable takes a patch that might break these older LTS branches that have yet to go out of support might be worthwhile.  We'll find such cases either way in Android though.

**Comment by nathanchance** [comment]
Fair enough, I thought that the most recent version of Android had kernels compiled with `clang-12` but I guess there could be other targets or older branches that matter? What branches should we support here with `clang-10` and what kind of build targets are interesting?
